python/ray/util/accelerators/accelerators.py

```diff
@@ -7,6 +7,7 @@
 NVIDIA_TESLA_A10G = "A10G"
 INTEL_MAX_1550 = "Intel-GPU-Max-1550"
 INTEL_MAX_1100 = "Intel-GPU-Max-1100"
+INTEL_GAUDI = "Intel-GAUDI"
```

Comment:
Can you also add INTEL_GAUDI2 here?

Reply:
Sure. I've kept it generic for now. Lets update post closure of the right instance usage ?

Reply:
Hi all, I'm working on LLM serving on Gaudi 2. Is Gaudi 2 not supported yet?